cmd/k8s-operator/sts.go

```diff
@@ -524,13 +528,6 @@ func (a *tailscaleSTSReconciler) reconcileSTS(ctx context.Context, logger *zap.S
 			},
 		},
 	}
-	for key := range configs {
-		fn := kubeutils.TailscaledConfigFileNameForCap(key)
-		configVolume.VolumeSource.Secret.Items = append(configVolume.VolumeSource.Secret.Items, corev1.KeyToPath{
-			Key:  fn,
-			Path: fn,
-		})
-	}
```

Comment:
Removed these after spending a fair bit of time trying to figure out how to test/fix tests. There is really no need to explicitly name them as we use the same names as Secret keys (and we have tests for Secret key names/contents)